ui/components/multichain/network-list-menu/network-list-menu.js

```diff
@@ -92,7 +92,7 @@ export const NetworkListMenu = ({ onClose }) => {
   const orderedNetworksList = useSelector(getOrderedNetworksList);
 
   const newOrderNetworks = () => {
-    if (orderedNetworksList.length === 0) {
+    if ((orderedNetworksList?.length ?? 0) === 0) {
```

Comment:
I know this is effective but it's a bit ugly.  The `?? 0` shouldn't be needed since it would otherwise return `undefined`.  A few thoughts:

1.  Can `getOrderedNetworksList` return an empty array?  
2.  Can `getOrderedNetworksList` simply return `nonTestNetworks` if it's empty?